docs/install/install-knative-gcp.md

```diff
@@ -14,180 +16,67 @@
-      native GCP resources.
-
-      You need to create a new Google Cloud Service Account named
-      `cloud-run-events` with the following command:
-
-      ```shell
-      gcloud iam service-accounts create cloud-run-events
-      ```
-
-      Then, give that Google Cloud Service Account permissions on your project.
-      The actual permissions needed will depend on the resources you are
-      planning to use. The Table below enumerates such permissions:
-
-      |    Resource / Functionality     |                                     Roles                                      |
-      | :-----------------------------: | :----------------------------------------------------------------------------: |
-      |        CloudPubSubSource        |                              roles/pubsub.editor                               |
-      |       CloudStorageSource        |                              roles/storage.admin                               |
-      |      CloudSchedulerSource       |                           roles/cloudscheduler.admin                           |
-      |      CloudAuditLogsSource       | roles/pubsub.admin, roles/logging.configWriter, roles/logging.privateLogViewer |
-      |        CloudBuildSource         |                            roles/pubsub.subscriber                             |
-      |             Channel             |                              roles/pubsub.editor                               |
-      |        PullSubscription         |                              roles/pubsub.editor                               |
-      |              Topic              |                              roles/pubsub.editor                               |
-      | Workload Identity in Data Plane |                         roles/iam.serviceAccountAdmin                          |
-
-      In this guide, and for the sake of simplicity, we will just grant
-      `roles/owner` privileges to the Google Cloud Service Account, which
-      encompasses all of the above plus some other permissions. Note that if you
-      prefer finer-grained privileges, you can just grant the ones described in
-      the Table. Also, you can refer to
-      [managing multiple projects](../install/managing-multiple-projects.md) in
-      case you want your Google Cloud Service Account to manage multiple
-      projects.
-
-      ```shell
-      gcloud projects add-iam-policy-binding $PROJECT_ID \
-        --member=serviceAccount:cloud-run-events@$PROJECT_ID.iam.gserviceaccount.com \
-        --role roles/owner
-      ```
-
-      Then, you can either:
-
-      - Use **Workload Identity**.
+## Install the Knative-GCP Constructs
+
+### Option 1: Install from Master using [ko](http://github.com/google/ko)
+
+```shell
+ko apply -f ./config
+```
+
+### Option 2: Install a [release](https://github.com/google/knative-gcp/releases).
+
+Remember to update `{{< version >}}` in the commands below with the
```

Comment:
Can we change version to a shell variable? So that users set it once and then they can copy-paste the other commands.

Reply:
Good call. Done.